app/channel.js

```diff
@@ -20,6 +20,16 @@ exports.channel = () => {
   return channel
 }
 
+exports.formattedChannel = () => {
```

Comment:
Existing method was left alone since telemetry also uses it (and the values in that case shouldn't change)